src/registrar/tests/common.py

```diff
@@ -161,7 +162,7 @@ def dummy_user(self, item_name, short_hand):
         user = User.objects.get_or_create(
             first_name="{} first_name:{}".format(item_name, short_hand),
             last_name="{} last_name:{}".format(item_name, short_hand),
-            username="{} username:{}".format(item_name, short_hand),
+            username="{} username:{}".format(item_name, str(uuid.uuid4())[:8]),
```

Comment:
After the edit on Contact.save, this dude breaks unless we make sure it's unique